client/Elements/CreateTopicWizardWithMoreOptions/CreateTopicsWizard.patternfly.css

```
@@ -0,0 +1,62 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+.topics-wizard-content {
+  max-width: 500px;
+}
+
+.pf-c-form__group-control > * {
+  margin-bottom: 16px;
```

Use patternfly var for spacing ie:
```suggestion
margin-bottom: var(--pf-global--spacer--md); 
```